README.md

```diff
@@ -1,2 +1,41 @@
 # LASAGNE-Core
 Layered Approach to Service Architectures for a Global Network Environment (LASAGNE)
+
+## About LASAGNE
+LASAGNE is a pattern-based framework for parallel and distributed computing
+* Multi-threaded distributing computing platform in C++
+* Extension mechanisms supported on a loadable, configurable infrastructure
```

Comment:
Maybe also add that LASAGNE adopts a Service Oriented Architecture (SOA) principals and behaviour each supported through meta-programming techniques